06_symbology_functions.sql

```diff
@@ -392,6 +392,9 @@ BEGIN
     WHEN TG_OP = 'DELETE' THEN
       rp_obj_id = OLD.obj_id;
   END CASE;
+  
+  UPDATE qgep.od_reach
```

Comment:
Can you add a comment here that explains why this is done (i.e. so we know in 5 years still, that this is done to retrigger the `calculate_reach_length` trigger on update)